other-file.txt

```diff
@@ -3,4 +3,4 @@ Congrats, you found the OTHER file!
 There's another question in this one, but I promise it's not too tricky!
 
 Question: What is your favourite song?
-Your answer:
\ No newline at end of file
+Your answer: Another World- Gojira
```

Comment:
My favourite song

Reply:
Nice choice Ali!

Reply:
Great!

Reply:
Explain what is this?

Reply:
> Explain what is this?

Gojira is a Metal band ;D. Next time I will report you :D :D